app/scripts/controllers/permissions/specifications.js

```diff
@@ -50,10 +50,10 @@ const CaveatFactories = Object.freeze({
  * PermissionController.
  *
  * @param {{
- *   getIdentities: () => Record<string, Identity>,
+ *   getInternalAccounts: () => Record<string, import('@metamask/eth-snap-keyring').InternalAccount>,
```

Comment:
Check if can be imported in the top of the file

Reply:
I thought the type lives in `@metamask/keyring-api`? I didn't see it in `@metamask/eth-snap-keying`